.github/workflows/ci.common.yml

```diff
@@ -39,3 +39,6 @@ jobs:
 
       - name: Docker stop
         run: npm run docker:stop:ci -- --filter=${{ inputs.package_name }}
+
+      - name: Run lint
```

Comment:
previously automerge would still happen if linting failed, now this shouldn't be the case